botfront/imports/api/storyGroups/storyGroups.collection.js

```diff
@@ -22,16 +21,13 @@ StoryGroups.deny({
 
 Meteor.startup(() => {
     if (Meteor.isServer) {
-        StoryGroups._ensureIndex({ projectId: 1, name: 1 }, { unique: true });
+        StoryGroups._ensureIndex({ projectId: 1, title: 1, name: 1 }, { unique: true });
```

Comment:
This was because it's possible to __dropIndex anytime, but apparently only possible to __ensureIndex while in Meteor.startup; if that is actually so, replacing the index with a previously non-existent key in a single migration is impossible. (Because for a short period every document will have that key _null_).

Reply:
but I'm following your idea to keep name so...